<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BeforeGame5Dialog</class>
 <widget class="QDialog" name="BeforeGame5Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>记者</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>你认为队长在场上的作用达到你的预期了吗？</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>641</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>您</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>530</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>他是一位优秀的队长，我很满意他的表现。</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>95</y>
      <width>530</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>他在比赛场上的表现很难称得上令我满意。</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>530</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>他很努力，但在担任队长一职上还需要锻炼。</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>205</y>
      <width>530</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>他能带领球队前进，是球队的精神领袖。</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>420</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>赛前新闻发布会</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
